docs/docs/sdk/js.md

```diff
@@ -3,12 +3,67 @@ id: js
 title: JavaScript
 ---
 
-To install the JavaScript SDK, run:
+This generator creates TypeScript/JavaScript client that utilizes
+[axios](https://github.com/axios/axios). The generated Node module can be used
+in the following environments:
+
+Environment
+
+- Node.js
+- Webpack
+- Browserify
+
+Language level
+
+- ES5 - you must have a Promises/A+ library installed
+- ES6
+
+Module system
+
+- CommonJS
+- ES6 module system
+
+It can be used in both TypeScript and JavaScript. In TypeScript, the definition
+should be automatically resolved via `package.json`.
+([Reference](http://www.typescriptlang.org/docs/handbook/typings-for-npm-packages.html))
+
+### Building
+
+To build and compile the typescript sources to javascript use:
+
+```
+npm install
+npm run build
+```
+
+### Publishing
+
+First build the package then run `npm publish`
+
+### Consuming
+
+navigate to the folder of your consuming project and run one of the following
+commands.
+
+_published:_
 
 ```
-npm install --save @oryd/hydra-client
+npm install @oryd/hydra-client@v1.9.0-alpha.2 --save
```

Comment:
If this is added, you need to update this tag on every new release. This can be automated using something like this: https://github.com/ory/hydra/blob/master/docs/config.js#L11-L16